resource_processor/resources/commands.py

```diff
@@ -48,53 +48,65 @@ async def build_porter_command(config, logger, msg_body, custom_action=False):
             # if still not found, might be a special case
             # (we give a chance to the method above to allow override of the special handeling done below)
             else:
-                parameter_value = get_special_porter_param_value(config, parameter_name, msg_body)
+                parameter_value = get_special_porter_param_value(
```

Comment:
There are lots of formatting changes in this PR.
Did the linter miss these previously, or has something else changed?
No problem with fixing up linting/formatting issues, but it makes this PR look bigger than it is.

Reply:
There's something not quite right in the config as the VS Code format command doesn't seem to apply the same formatting for me as is used in the code. Something to look into at some point. 
For now, I've reverted the formatting-only changes that snuck in